src/inputs/NumberField.tsx

```diff
@@ -120,7 +120,7 @@ export function NumberField(props: NumberFieldProps) {
       // This is called on each DOM change, to push the latest value into the field
       onChange={(rawInputValue) => {
         // If the wip value is invalid, i.e. it's `10b`, don't push that back into the field state
-        const wip = Number((rawInputValue || "").replace(/[^0-9\.]/g, ""));
+        const wip = Number((rawInputValue || "").replace(/[^-0-9\.]/g, ""));
```

Comment:
Thanks for fixing, could you also please add a test?

Reply:
Thanks for adding. Very odd that the test passed for negative numbers without the change you made. Makes me want to look into why that'd happen.

Reply:
I'm thinking of changing this to just `wip = parseFloat(rawInputValue)` instead of using regex. Any objections to that?